main.dart

```diff
@@ -270,7 +308,7 @@ Future<void> createFunction() async {
   print('Running Create Function API');
   try {
     final res = await functions.create(
-        functionId: 'testfunction',
+        functionId: ID.custom('testfunction'),
```

Comment:
If we use unique here we can run the playground multiple times without issue

```suggestion
        functionId: ID.unique(),
```

Reply:
Updated!